models/user.go

```diff
@@ -719,6 +720,18 @@ func (u *User) IsMailable() bool {
 	return u.IsActive
 }
 
+// EnabledEmailNotifications checks if the user has
+// enabled receiving notifications by email
+func (u *User) EnabledEmailNotifications() bool {
```

Comment:
Why not just add this condition to IsMailable?

Reply:
I was thinking that maybe if other email notifications are added, we'd want a distinction between whether the user chose not to receive emails or whether they are not active. So for example, security related emails get sent regardless of the user's email notification choice but doesn't get sent if the user is not active.